packages/flutter/test/foundation/service_extensions_test.dart

```diff
@@ -173,7 +173,7 @@ void main() {
     const int disabledExtensions = kIsWeb ? 2 : 0;
     // If you add a service extension... TEST IT! :-)
     // ...then increment this number.
-    expect(binding.extensions.length, 35 + widgetInspectorExtensionCount - disabledExtensions);
+    expect(binding.extensions.length, 36 + widgetInspectorExtensionCount - disabledExtensions);
```

Comment:
nit: move 36 into a const so what it represents is better documented, since it isn't introduced in the test itself.

Reply:
done.